src/templates/bespoke/presenter/presenter-view.tsx

```diff
@@ -73,9 +93,26 @@ const presenterView = (deck) => {
     return $cache[klass]
   }
 
+  const resizeNotes = (direction: string) => {
+    const current = parseFloat($(classes.noteContainer).style.fontSize) || 1
+    const intended = current + (direction === 'bigger' ? 0.1 : -0.1)
+    $(classes.noteContainer).style.fontSize = `${intended.toFixed(1)}em`
+  }
+
   const subscribe = (deck) => {
     // Next slide view
     $(classes.nextContainer).addEventListener('click', () => deck.next())
+    $(classes.noteButtonsBigger).addEventListener('click', () =>
+      resizeNotes('bigger')
+    )
+    $(classes.noteButtonsSmaller).addEventListener('click', () =>
+      resizeNotes('smaller')
+    )
+
+    document.addEventListener('keyup', (e: KeyboardEvent) => {
+      if (e.key === '+' || e.key === '=') resizeNotes('bigger')
+      if (e.key === '-' || e.key === '_') resizeNotes('smaller')
```

Comment:
`=` and `_` are not required.

In my keyboard layout, there is no `_` key assign in `-` keycap. Instead, `-` key has `=` key assign.

Reply:
On my uk keyboard it looks like this. The plus requires holding shift.
The extra key binding while not correct felt more intuitive
![image](https://user-images.githubusercontent.com/715120/154810377-601ac410-9666-4adc-94c6-1f7abc6f8901.jpeg)

Reply:
I think it is not intuitive for me because my key layout will trigger unexpected scaling by not related key.

![IMG_2531](https://user-images.githubusercontent.com/3993388/154813309-42f40f85-da92-4431-a23d-92898c3d39c1.JPG)